common/djangoapps/student/management/commands/delete_historical_enrollment_data.py

```diff
@@ -1,31 +1,4 @@
 """
-Command to delete all rows from the student_historicalcourseenrollment table.
+We kept this file as an example to show how to use row_delete.py to delete rows from a table using best practices.
+It would be safe to delete this file if you used row_delete.py somewhere else and updated README_row_delete.rst
 """
-
```

Comment:
@doctoryes @bmedx I'm not sure that this is what you hand intended? I'm not sure this file with a comment is really enough to let the reader know how to do this. @mahyard I believe the intent here was to keep the code example as well but will wait for others to weigh in.

Reply:
@macdiesel This trick is to save the file and its history while reducing the size as much as possible. any one who intent to use this file can go through the history and find different versions of it. In addition we get rid of putting time to maintain the code e.g. when upgrading python version.

Reply:
I agree with the removal of unused code to remove maintenance problems. IMO the appropriate place for the example usage code is in this file:
https://github.com/edx/edx-platform/blob/41192041285dc423a1017974a53c73d52313d1b4/openedx/core/djangoapps/util/row_delete.py

How about the following steps to get this PR merged?:
- Add the `from` imports and the `class Command()` code to the docstring in the file above, with a short explanation that this is an example of usage.
- Delete this file.
- Delete the `README_row_delete.rst` file.